include/Reader/reader.h

```diff
@@ -47,12 +47,7 @@
 // Debugging
 // -----------------------------------------------------------------
 
-#define ASSERTM(Predicate, Message)                                            \
-  do {                                                                         \
-    if (!(Predicate)) {                                                        \
-      ReaderBase::debugError(__FILE__, __LINE__, Message);                     \
-    }                                                                          \
-  } while (0)
+#define ASSERTM(Predicate, Message) assert((Predicate) && Message)
```

Comment:
We really might as well just go ahead and inline and remove this macro at this point...